src/main/scala/rules/SymbolicExecutionRules.scala

```diff
@@ -48,6 +49,10 @@ trait SymbolicExecutionRules {
         res.counterexample = Some(finalCE)
       }
     }
+    if(Verifier.config.enableBranchconditionReporting() && v.decider.pcs.branchConditionExps.nonEmpty){
+        res.branchConditions = Seq(v.decider.pcs.branchConditionExps
+          .reduce((e1: Exp, e2: Exp) => And(e1,e2)(pos = NoPosition, info = NoInfo, errT = NoTrafos)))

```

Comment:
https://github.com/viperproject/silicon/blob/master/src/main/scala/Utils.scala#L115 instead of `reduce`?